### PR TITLE
Sync fixes

### PR DESCRIPTION
**Pull request by javiserrano**

### CODE REVIEW

File: src/megaclient.cpp [hunk 1]
```diff
@@ -9913,6 +9913,15 @@ bool MegaClient::syncup(LocalNode* l, dstime* nds)
                                 RegCloseKey(hKey);
                             }
                         }
+
+                        lpath.append((char *)L":OECustomProperty", 34);
```
Comment (sergiohs84) [inline src/megaclient.cpp]: No idea about the purpose of this tailing string for the filename

File: src/node.cpp [hunk 2]
```diff
@@ -1073,8 +1073,11 @@ void LocalNode::setnameparent(LocalNode* newparent, string* newlocalpath)
         }
     }
 
-    LocalTreeProcUpdateTransfers tput;
-    sync->client->proclocaltree(this, &tput);
+    if (newlocalpath)
```
Comment (sergiohs84) [inline src/node.cpp]: Why this new restriction?

File: src/megaclient.cpp (hunk 1 above)
Comment (javiserrano) [inline src/megaclient.cpp]: Windows Search (enabled by default on Windows) adds this ADS to .eml and .pst files just after they are downloaded. Unfortunately that changes the modification time of the files too and Windows Search doesn't restore it.

File: src/node.cpp (hunk 2 above)
Comment (javiserrano) [inline src/node.cpp]: If the path of the LocalNode hasn't changed, there is no need to update related Transfers. Also, that can cause crashes in some corner cases.

File: src/win32/fs.cpp [hunk 3]
```diff
@@ -660,9 +663,35 @@ bool WinFileSystemAccess::istransientorexists(DWORD e)
...
+    int r = 0;
+#ifndef WINDOWS_PHONE
+    for (set<WinDirNotify*>::iterator it = dirnotifys.begin(); it != dirnotifys.end(); it++)
+    {
+        if ((*it)->enabled)
+        {
+            DWORD bytes = 0;
+            if (GetOverlappedResult((*it)->hDirectory, &((*it)->overlapped), &bytes, FALSE))
+            {
+                r = 1;
```
Comment (sergiohs84) [inline src/win32/fs.cpp]: Use `Waiter::NEEDEXEC` better than `1`

**Pull request merged**